# Author: Damyan Ivanov <dmn@debian.org>
# Description: fix POD errors
#              missing =over and =back's
# Upstream-Bug: http://rt.cpan.org/Public/Bug/Display.html?id=42021
--- a/Strip.pm
+++ b/Strip.pm
@@ -158,6 +158,8 @@ between calls to $hs->parse().
 
 =head2 METHODS
 
+=over
+
 =item new()
 
 Constructor. Can optionally take a hash of settings (with keys
@@ -201,6 +203,8 @@ any conversion of tags into spaces. Set 
 Takes a boolean value. If set to false, HTML::Strip will decode HTML
 entities. Set to true by default.
 
+=back
+
 =head2 LIMITATIONS
 
 =over 4
@@ -227,6 +231,8 @@ excess whitespace (for example, using C<
 HTML::Strip will only attempt decoding of HTML entities if
 L<HTML::Entities> is installed.
 
+=back
+
 =head2 EXPORT
 
 None by default.
